serverless/pages/saved-objects.mdx

```diff
@@ -19,7 +19,7 @@ TBD: Need serverless-specific RBAC requirements
 
 To access **Saved Objects**, you must have the required `Saved Objects Management` ((kib)) privilege.
 
-To add the privilege, open the main menu, and then click **Stack Management → Roles**.
+To add the privilege, open the main menu, and then click **Management → Roles**.
```

Comment:
Should this be `Custom Roles`?
```suggestion
To add the privilege, open the main menu, and then click **Management → Custom Roles**.
```

Also, FWIW, in serverless Security the nav menu calls this `Stack Management` (while Search and Obs call it `Management`). Should we even try to accommodate such granular differences? Or maybe adopt a more nav-agnostic approach like we're thinking with the ESS nav changes (more generic instructions like "navigate to the Management page," remind users to use the search box, etc.)?

<img width="1589" alt="image" src="https://github.com/user-attachments/assets/231d0a2d-d59f-4403-a18f-5be07854f6d9">

Reply:
More alignment would be fab! Seems fair to anticipate that, helps us keep things simple.

Reply:
Ah thanks, I missed the comment tags originally.